dispatcher/conf/dispatcher.any

```diff
@@ -48,9 +48,9 @@
       /rend01
         {
         # Hostname or IP of the render
-        /hostname "aem-hostname"
+        /hostname "mhaack-macbook.local"
```

Comment:
Replace with  a generic placeholder string

Reply:
Arg, yes I will revert that was from my local testing :-)